py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -198,25 +198,18 @@ def compile(
     gm = apply_lowering_passes(gm, torch_inputs)
     logger.debug("Lowered Input graph: " + str(gm.graph))
 
-    enabled_precisions = set(enabled_precisions)
-
-    if (
-        torch.float16 in enabled_precisions
-        or torch_tensorrt.dtype.half in enabled_precisions
-    ):
-        precision = torch.float16
-    elif (
-        torch.float32 in enabled_precisions
-        or torch_tensorrt.dtype.float in enabled_precisions
-    ):
-        precision = torch.float32
+    if dtype.float16 in enabled_precisions or dtype.half in enabled_precisions:
```

Comment:
`dtype.float16` and `dtype.half` seem to point to the same `enum` object. Should this be:

```python
if dtype.float16 in enabled_precisions or torch.float16 in enabled_precisions:
```